contrib/cirrus/mac_setup.sh

```diff
@@ -31,3 +31,10 @@ echo "TMPDIR=/private/tmp/ci" >> $CIRRUS_ENV
 
 # Removed completely during cleanup.
 mkdir -p /private/tmp/ci
+
+# Some test operations & checks require a git "identity"
+_gc="git config --file $HOME/ci/.gitconfig"
```

Comment:
Even though the `HOME=$HOME/ci` trick is only five lines up, please add a comment here explaining this. Maybe something like one of these
```
# $HOME in this context does not include the /ci part (see above)
# CI process will include /ci in its $HOME, but in this context it does not
# See above for hackery redefining $HOME
```

Reply:
Good idea, I thought that might be easily missed.  Nothing escapes your sharp eyes though :+1:

Reply:
It's precisely **because** the $HOME trickery escaped my eyes that I requested the comment! :grin: